test/kitchen/test/integration/common/rspec_datadog/spec_helper.rb

```diff
@@ -448,14 +448,18 @@ def fetch_python_version(timeout = 15)
 def is_file_signed(fullpath)
   puts "checking file #{fullpath}"
   expect(File).to exist(fullpath)
+  
   output = `powershell -command "(get-authenticodesignature -FilePath '#{fullpath}').SignerCertificate.Thumbprint"`
-  ## signature below is for new cert acquired May 2023 using new hsm-backed signing method
-  signature_hash = "B03F29CC07566505A718583E9270A6EE17678742"
-  if output.upcase.strip == signature_hash.upcase.strip
-    return true
-  end
+  signature_hashes = {
+    ## signature below is for new cert acquired May 2023 using new hsm-backed signing method
+    "B03F29CC07566505A718583E9270A6EE17678742".upcase.strip => true,
+  }
 
-  puts("expected hash = #{signature_hash}, actual hash = #{output}")
+  if signature_hashes.key?(output.upcase.strip)
+    return true    
+  end
```

Comment:
May I suggest we use a [set](https://ruby-doc.org/stdlib-2.7.1/libdoc/set/rdoc/Set.html) here instead? Something like

```ruby
  signature_hashes = Set[
    ## signature below is for new cert acquired May 2023 using new hsm-backed signing method
    "B03F29CC07566505A718583E9270A6EE17678742".upcase.strip,
  ]

  return true if signature_hashes.include?(output.upcase.strip)

```

Reply:
It seems a lot more idiomatic. Thansk for the suggesiton. Added